### ISSUE TITLE
Tablet controls

### ISSUE THREAD

**Issue by xarx00** [doukutsu-rs/doukutsu-rs#274]
On Android (13 &14), the left arrow controller is too big for a tablet. The game is playable on phones, but hard to play on tablets.

Would it be possible to reduce the size of the arrow controller for tablets? I haven't found any settings that would allow that.

**Comment by Persvicx** [comment]
Yeah.. I have a 11 inch tablet and its impossible to play without using samsung's window  mode and shrinking the window vertically
